### PR TITLE
Feat/bump jsonld

### PR DESCRIPTION
**Pull request by lemoustachiste**
hi @gjgd,

I have bumped the version. I couldn't align with jsonld@6 because of an ESM issue with a digital bazaar package.
I had to update the tests to include a `documentLoader` for the various contexts used in the tests.
I did not bump the package.json version, I am not sure what should be the next version (semver or not?).

This fixes the main issue in #91 

Thanks

### CODE REVIEW

File: packages/lib/src/test/jsonld.test.ts
Comment (gjgd) [inline packages/lib/src/test/jsonld.test.ts:24]: Can you move tests with a custom document loader to this file? https://github.com/gjgd/jsonld-checker/blob/main/packages/lib/src/test/customDocumentLoader.test.ts

**Review by gjgd — COMMENTED** [review]
LGTM overall, just one comment

File: packages/lib/src/test/jsonld.test.ts
Comment (lemoustachiste) [reply packages/lib/src/test/jsonld.test.ts:24]: Hm, I'm not sure it's the same thing, and maybe I should have used a better name for it.
Here it's really a helper to preload the various contexts needed for the jsonld test to pass. Without that I would be getting errors: "context could not be dereferenced to a valid jsonld file" or something along these lines. 

Here the helper file: https://github.com/gjgd/jsonld-checker/pull/95/files#diff-b9fc4027149f6ba7e0fae4648dc59aff487e09f9bdb457776d1541a901dd5b6e. I could rename it (and its usage) to something less confusing if you want.

**Comment by lemoustachiste** [comment]
@gjgd I have updated the tests to use the document loader as needed.
I have also had to update the linting configuration as per the error showing up in this run: https://github.com/lemoustachiste/jsonld-checker/runs/7563878410?check_suite_focus=true

**Review by gjgd — APPROVED** [review]

**Comment by gjgd** [comment]
Thanks for the PR! Are you able to merge it after I approve or do you need me to do it?

**Comment by lemoustachiste** [comment]
Nope, can't merge it.
Do you handle publishing too?

**Comment by gjgd** [comment]
Just published a new patch version: https://www.npmjs.com/package/jsonld-checker

Can you check and see if it works?

**Pull request merged**
